src/magicbox/InputManager.ts

```diff
@@ -218,14 +218,15 @@ export class InputManager {
   private keydown(e: KeyboardEvent) {
     switch (e.keyCode || e.which) {
       case 9: // Tab key
```

Comment:
Maybe it would be possible to use `KEYBOARD.TAB` here similar to the unit tests, and to remove the comment?